ml-agents-envs/mlagents/envs/sampler_class.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np 
+from abc import ABC, abstractmethod
+
+from .exception import SamplerException
+
+class Sampler(ABC): 
+
+    @abstractmethod
+    def sample_parameter(self, *args, **kwargs):
+        pass
+
+
+class UniformSampler(Sampler):
+    def __init__(self, param_dict):
+        self.check_keys(param_dict)
+        self.min_value = param_dict["min"]
+        self.max_value = param_dict["max"]
+
+
+    def check_keys(self, param_dict):
```

Comment:
The other advantage of passing the min and max as params to init is that you don't need this check.